views/courses/edit-course.hbs

```diff
@@ -133,6 +133,8 @@
 							</option>
 						{{/each}}
 					</select>
+					<span class="small" id="courseTeacherErr" style="visibility:hidden; color: #ff1134;">
```

Comment:
color property in the html. And what if we want to change the color globaly?

Reply:
done